dfibert/envs/RLTractEnvironment_fast.py

```diff
@@ -108,8 +108,8 @@ def __init__(self, device, seeds=None, step_width=0.8, dataset='100307', grid_di
 
             fa_img = dti_fit.fa
             seed_mask = fa_img.copy()
-            seed_mask[seed_mask >= 0.2] = 1
-            seed_mask[seed_mask < 0.2] = 0
+            seed_mask[seed_mask >= self.fa_threshold] = 1
```

Comment:
`self.fa_threshold` is the threshold to stop tracking at low FA values. The seeds should actually be picked at places with high fa values (=low ambiguity).

Reply:
fixed in commit 9b690b4